### ISSUE TITLE
Review: 2.4.12 Focus Not Obscured (Enhanced) (AAA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#60]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 2.4.12 considering the mobile research questions (to be completed)

---
When a user interface component receives keyboard focus, no part of the component is hidden by author-created content.
---
Sufficient Techniques for Success Criterion 2.4.12

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)

    [C43: Using CSS scroll-padding to un-obscure content ](https://www.w3.org/WAI/WCAG22/Techniques/css/C43.html)

Failures for Success Criterion 2.4.12

    An interaction that causes content to appear over the component with keyboard focus, visually covering part of the focus indicator. This behavior might be encountered with advertising or promotional material meant to provide more information about a product as the user navigates through a catalogue.
    A page has a sticky footer (attached to the bottom of the viewport). When tabbing down the page, a focused item is partially hidden by the footer because content in the viewport scrolls without sufficient [scroll padding](https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding).
